client/app/components/account_config.coffee

```diff
@@ -46,8 +46,8 @@ module.exports = React.createClass
         nstate.isWaiting         = RouterStore.isWaiting()
```

Comment:
It seems that `_newAccountWaiting` has been replaced by `RouterGetter.getRequestStatus`; please remove old methods that you have replaced (for good! :) such as `isWaiting()`, `isChecking()`, `_newAccountWaiting`, `_newAccountChecking`, `_serverAccountErrorByField` etc.